### PR TITLE
fix(lambda)!: bump timeout and memory size based on failure data

### PR DESCRIPTION
**Pull request by aping1**
We have been seeing a number of lambda snapshot failures in prod. These failures were tracked down to larger snapshots taking longer than 60 seconds. We are bumping the timeout to 120 for slower failures but more reliable first runs
https://github.com/observeinc/cloudformation-aws-collection/pull/24
## Testing
``` # module.observe.module.observe_lambda.aws_lambda_function.this will be updated in-place
  ~ resource "aws_lambda_function" "this" {
        id                             = "testytest-observe-allie"
      ~ memory_size                    = 128 -> 256
        tags                           = {}
      ~ timeout                        = 60 -> 120
        # (22 unchanged attributes hidden)

        # (3 unchanged blocks hidden)
    }```

### CODE REVIEW

**Review by obs-gh-irismcleary — APPROVED** [review]

**Review by bendrucker — CHANGES_REQUESTED** [review]
Changing the default memory is a breaking change, as it causes materially different behavior for existing callers. So this should be `fix!(lambda): ...`. 

Also, the readme is out of sync, make sure to `pre-commit run -a` locally to generate that.

**Comment by bendrucker** [comment]
Actually `fix(lambda)!:

https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-scope-and--to-draw-attention-to-breaking-change

**Comment by aping1** [comment]
Ok autogenerated the readmet with `pre-commit run -a`

File: variables.tf [hunk 1]
```diff
@@ -38,7 +38,7 @@ variable "lambda_memory_size" {
   EOF
   type        = number
   nullable    = false
-  default     = 128
+  default     = 256
```
Comment (bendrucker) [inline variables.tf:41]: The `description` (see above) makes reference to this and needs updating.

**Pull request merged**